platform/ios/PolyPodApp/PolyPod.xcodeproj/project.pbxproj

```diff
@@ -91,14 +91,27 @@
 		3B3CEDFA274FC37C000C3E6E /* FilePicker.swift in Sources */ = {isa = PBXBuildFile; fileRef = E989B350269CA6B7000EE977 /* FilePicker.swift */; };
 		3B5487E227CD0FF700480CB5 /* Endpoint.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3B5487E127CD0FF700480CB5 /* Endpoint.swift */; };
 		5B8653022643FC310014208C /* PolyNav.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5B8653012643FC310014208C /* PolyNav.swift */; };
+		9015F60927EB0F7900DC6142 /* Core.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9015F60827EB0F7900DC6142 /* Core.swift */; };
+		9015F60A27EB0F7900DC6142 /* Core.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9015F60827EB0F7900DC6142 /* Core.swift */; };
+		9015F60C27EB103C00DC6142 /* PolyPodCore.xcframework in Frameworks */ = {isa = PBXBuildFile; fileRef = 9015F60427EB0DA500DC6142 /* PolyPodCore.xcframework */; };
+		9015F60D27EB103C00DC6142 /* PolyPodCore.xcframework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 9015F60427EB0DA500DC6142 /* PolyPodCore.xcframework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
 		901C0588278EE58700BC8364 /* CoreDataStack.swift in Sources */ = {isa = PBXBuildFile; fileRef = 901C0587278EE58700BC8364 /* CoreDataStack.swift */; };
 		901C058A278F2AF800BC8364 /* CoreData+PolyIn.swift in Sources */ = {isa = PBXBuildFile; fileRef = 901C0589278F2AF800BC8364 /* CoreData+PolyIn.swift */; };
 		901C058B278F2D4A00BC8364 /* CoreData+PolyIn.swift in Sources */ = {isa = PBXBuildFile; fileRef = 901C0589278F2AF800BC8364 /* CoreData+PolyIn.swift */; };
 		901C058C278F34A800BC8364 /* CoreDataStack.swift in Sources */ = {isa = PBXBuildFile; fileRef = 901C0587278EE58700BC8364 /* CoreDataStack.swift */; };
 		9049ECA427BCE8B70059605A /* CocoaLumberjackSwift in Frameworks */ = {isa = PBXBuildFile; productRef = 9049ECA327BCE8B70059605A /* CocoaLumberjackSwift */; };
 		904D4A7027905D5B00A75AED /* DispatchToMainQueue.swift in Sources */ = {isa = PBXBuildFile; fileRef = 904D4A6F27905D5B00A75AED /* DispatchToMainQueue.swift */; };
 		904D4A712790633000A75AED /* DispatchToMainQueue.swift in Sources */ = {isa = PBXBuildFile; fileRef = 904D4A6F27905D5B00A75AED /* DispatchToMainQueue.swift */; };
+		9063A7B727EB6DE600955E49 /* FlatBuffers in Frameworks */ = {isa = PBXBuildFile; productRef = 9063A7B627EB6DE600955E49 /* FlatBuffers */; };
+		9063A7E527EC637000955E49 /* core_bootstrap_response_generated.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9063A7E127EC637000955E49 /* core_bootstrap_response_generated.swift */; };
```

Comment:
This how Xcode will link the generated FlatBuffer models. In Current state when new models are added or removed this file will have to be updated. It would have been nice to ignore these lines, but I would not want to go into ignoring lines from this core file.

I will experiment with encapsulating the files in a Swift Package, this way, this file should be left clean without any knowledge of the generated files.




 It would have been nice to

Reply:
Created PolyPodCoreSwift Package which is will facilitate using Rust's core in Swift environment.